dimagi/commcare-hq
# Attendance Tracking: Attendees

## Technical Summary

Context: [Spec](https://docs.google.com/document/d/1hU2SQvnfkfxu3RoxhRCKL6IuWl6zO7nuZFEbgF5grrE/edit?pli=1#heading=h.whks7x3gtv74)

Adds UI to list expected attendees for an Attendance Tracking Event.

![image](https://user-images.githubusercontent.com/708421/223602130-a3ed5717-e48d-4e9c-8adb-5e1b8aaaea7d.png)

I kept "Add `remove_event()` view" and "Drop `remove_event()` view" commits so that we can revert the "Drop" commit if we want that option. It might be useful to be able to delete an Event under limited circumstances?

:blowfish: 

## Feature Flag

Attendance Tracking

## Safety Assurance

### Safety story

Tested locally

### Automated test coverage

Includes tests

### QA Plan

QA planned when feature is more mature.

See comment [here](https://github.com/dimagi/commcare-hq/pull/32705#issuecomment-1463661423).

### Rollback instructions




### Labels & Review
